arch/Kconfig

```diff
@@ -80,6 +80,42 @@ config BIG_ENDIAN
          is used to select linker script OUTPUT_FORMAT and command
          line option for gen_isr_tables.py.
 
+if ARC || ARM || NIOS2 || X86
+
+config SRAM_SIZE
+	int "SRAM Size in kB"
```

Comment:
I assume that this is unrelated to the PR, so no requirement to reply, but for the record;

I believe we should generally use bytes everywhere, and only translate to other units when necessary (e.g. when presenting human readable data).

Reply:
Also, when not using the "standard" (context-dependent) unit, one should include the unit information in the symbol;

so SRAM_SIZE_IN_KB.

But I assume that CONFIG_SRAM_SIZE is a pre-existing concept in some form and that changing this is out-of-scope for this PR.

Reply:
Agreed, its needs cleaning up across the code base.

Reply:
Yes, that change is outside of the scope of this PR.